capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -1,59 +1,60 @@
 import React, { Component } from 'react';
 import { Button, Form, Row } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
-import { UserCard } from './UserCard'
-
-
+import { UserCard } from './UserCard';
+import TextField from '@material-ui/core/TextField';
 import '../styles/Groups.css';
 
+
 class AdminClubPage extends Component {
   constructor(props) {
     super(props);
     this.state = {
       club: {},
       requesters: [],
+      members: [],
     }
   }
 
   fetchData = () => {
     fetch(`/api/clubs?id=${this.props.match.params.id}`)
       .then(response => response.json()).then(clubs => {
         const club = clubs[0];
-        this.setState({ club, requesters: [] });
+        this.setState({ club, requesters: [], members: [] });
         Promise.all(club.requestIDs.map(r => {
           return fetch(`/api/user?id=${r}`)
             .then(response => response.json())
             .then(member => member && this.setState({ requesters: [...this.state.requesters, member] }))
-            .catch(function (err) {
-              //TODO #61: Centralize error output
-              alert(err);
-            });
+            .catch(e => console.log(e));
```

Comment:
I don't think you need this catch here since it should jump to the .catch at the end of the series of statements on line 37? I may be wrong

Reply:
You're right! Will fix.